src/index.js

```diff
@@ -79,4 +80,4 @@ server.listen(port)
 server.on('error', onError)
 server.on('listening', onListening)
 
-console.log('server: gateway-api | port:', port, '| balancer-network:', balancerNetwork);
+console.log('server: gateway-api | port:', port, '| balancer-network:', balancerNetwork, '| uniswap-network:', uniswapNetwork);
```

Comment:
Remove this line, since #10 changes this to "ethereum chain"

Reply:
Done!